main/guides/orchestration/getting-started/api.md

```diff
@@ -0,0 +1,86 @@
+# Orchestration API
```

Comment:
Not sure if it's worth working on but I'm a very visual person so I like graphs. In my mind I kinda produced below graph that helped me mentally:

```
orchestrator
|
getChain()
|
v
chain
|
makeAccount()
|
v
orchestrationAccount (getAddress, getBalances, getBalance, etc)
|
uses under the hood
|
v
IcaAccount
```

Worth adding something like this graph(albeit a lot more nicely looking) at the top of this page?

Reply:
love this idea! @LuqiPan

Reply:
this is a great idea @LuqiPan, having a visual up top is impactful

Reply:
we've had pretty good luck with mermaid sequence diagrams

e.g. there are several in the [Trading with Offer Safety](https://docs.agoric.com/guides/zoe/contract-walkthru.html#trading-with-offer-safety) walkthru

Note that both the mermaid source and generated .svg are checked in
There's a nice vs-code extension for generating the .svg with one key